templates/cemanager.yaml

```diff
@@ -177,6 +177,9 @@ spec:
     - name: tcp-cemanager
       port: 8080
       protocol: TCP
+    - name: tcp-cemanager-https
```

Comment:
does the ingress config need to be modified to forward traffic to this port, too?

Reply:
Yeah so I'm in a bit of a weird situation rn with this...

This is to support https://github.com/cdr/m/pull/8248, which I guess is intended to only work after the ingress has been removed. 
So this helm change is only relevant for deployment configurations where ce-manager is directly accessible with an external IP and people want to access it over https without going through the ingress. 

TBH I'm a bit hesitant about both of these PRs because of their reliance on "future changes" 🤷 

Lmk what you think 🙏

Reply:
I think it's fine. We don't need to put this in the 1.20 changelog if we don't want customers to use it

Reply:
I agree with @deansheather, there's no harm in merging this, it just won't be useful unless people forward traffic to the pod through some means. If it makes your testing easier, then it makes sense to me to just merge this :)